### DISCUSSION TITLE
请问怎么设置默认软解

### DISCUSSION THREAD

**Discussion by deadpoolMIX** [hooke007/mpv_PlayKit#131] [Q&A 问答] [answered]
我的显卡时gtx950，一硬解画面就马赛克，每次都调有点麻烦，请问如何设置默认软解（小白提问）

**Comment by hooke007** [comment, marked as answer]
好歹翻翻mpv.conf的注释吧
https://github.com/hooke007/MPV_lazy/blob/781803cd0f606e8fd5718eeef14995c67341df53/portable_config/mpv.conf#L23

![image](https://user-images.githubusercontent.com/41094733/170081113-6cc9dbb8-ca8d-4e53-99db-17bef228773c.png)

**Reply by hooke007** [reply]
如果觉得 mpv-lazy 的门槛还是过高的话， mpv.net_CM 可能更适合

**Reply by deadpoolMIX** [reply]
一针见血了属于是，说实话我把input、mpv、profiles都看了，果然自己理解能力不太行
所以说，想要启动哪个就把哪个的#删掉保存是吗？

**Reply by deadpoolMIX** [reply]
搞好了，感谢，我下次会再读的仔细点的

**Reply by upyzl** [reply]
> 一针见血了属于是，说实话我把input、mpv、profiles都看了，果然自己理解能力不太行 所以说，想要启动哪个就把哪个的#删掉保存是吗？

非程序员/极客类用户首次接触估计是不好理解吧，我建议你可以装个notepad3（安装时默认取代记事本），看conf文件有语法高亮，就容易理解了

**Reply by deadpoolMIX** [reply]
> 

我还真是这类用户，不过我有学编程的意向，目前只是想把一些技术搞懂
